bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetEnergyHandler.java

```diff
@@ -210,4 +253,39 @@ private void updateActivePower(BaseOpenMessage msg) {
             updateState(CHANNEL_POWER, UnDefType.UNDEF);
         }
     }
+
+    /**
+     * Updates current day totalizer
+     *
+     * @param msg the EnergyManagement message received
+     * @throws FrameException
+     */
+    private void updateCurrentDayTotalizer(BaseOpenMessage msg) {
+        Double currentDayEnergy;
+        try {
+            currentDayEnergy = Double.parseDouble(msg.getDimValues()[0]) / 1000d;
+            updateState(CHANNEL_ENERGY_TOTALIZER_DAY, new QuantityType<Energy>(currentDayEnergy, Units.KILOWATT_HOUR));
+        } catch (FrameException e) {
+            logger.warn("FrameException on frame {}: {}", msg, e.getMessage());
+            updateState(CHANNEL_ENERGY_TOTALIZER_DAY, UnDefType.UNDEF);
+        }
+    }
+
+    /**
+     * Updates current month totalizer
+     *
+     * @param msg the EnergyManagement message received
+     * @throws FrameException
```

Comment:
To be removed, FrameException is not thrown but caught

Reply:
ok I would modify _updateActivePower()_ also for similarity